src/clients/HubPoolClient.ts

```diff
@@ -516,30 +531,33 @@ export class HubPoolClient extends BaseAbstractClient {
     );
   }
 
-  getTokenInfo(chainId: number | string, tokenAddress: string): L1Token | undefined {
-    const deposit = { originChainId: parseInt(chainId.toString()), inputToken: tokenAddress } as Deposit;
+  getTokenInfo(chainId: number | string, tokenAddress: Address): L1Token | undefined {
+    const deposit = {
+      originChainId: parseInt(chainId.toString()),
```

Comment:
Driveby: Should be able to drop the `toString()` here:

```
> y = 1
1
> parseInt(y)
1
> parseInt(y.toString())
1
```

```suggestion
      originChainId: parseInt(chainId.toString()),
```

Reply:
nvm - this is a tsc constraint, not a js constraint.